terraform/service_acc.tf

```diff
@@ -39,6 +39,6 @@ data "aws_iam_policy_document" "service_account" {
 }
 
 
-output "service_account_iam" {
-  value = data.aws_iam_policy_document.service_account.json
-}
+# output "service_account_iam" {
```

Comment:
lets not bloat our codebase with commented code

Reply:
Added comment to reason why